pkg/apis/ark/v1/constants.go

```diff
@@ -37,4 +37,8 @@ const (
 	// NamespaceScopedDir is the name of the directory containing namespace-scoped
 	// resource within an Ark backup.
 	NamespaceScopedDir = "namespaces"
+
+	// DefaultMetricAddress is the port at which the metric server will listen on to make the
```

Comment:
Let's call this DefaultMetricsAddress please. I'd also move the constant to `pkg/cmd/server/server.go`.